### PR TITLE
docs: add issue template and PR template

### PR DESCRIPTION
**Pull request by takeshi-iwanari**
Signed-off-by: takeshi.iwanari <takeshi.iwanari@tier4.jp>

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+## Pre-review checklist for the PR author
+
+- [ ] I've confirmed the [contribution guidelines](https://github.com/tier4/caret/blob/main/.github/CONTRIBUTING.md).
+- [ ] The PR is ready for review.
+
+## In-review checklist for the PR reviewers
+
+The PR reviewers **must** check the checkboxes below before approval.
+
+- [ ] The PR has been properly tested.
```
Comment (takam5f2) [inline .github/pull_request_template.md:22]: @takeshi-iwanari  
Which do you think is this checking item is required?

Comment (takeshi-iwanari) [reply .github/pull_request_template.md]: @takam5f2 
Thank you for pointing it out.
The check is unnecessary. I remove the line in the new commit.

Comment (takam5f2) [reply .github/pull_request_template.md]: @takeshi-iwanari  
Thank you. I think that the PR is acceptable.

**Review by takam5f2 — APPROVED** [review]
LGTM

**Pull request merged**